docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -87,6 +87,38 @@ To change alert statuses, either:
 * In the Alerts table, select all the alerts you want to change, and then select
 *Take action* -> *Close selected*, *Open selected*, or *Mark in progress*.
 
+[float]
+[[signals-to-cases]]
+=== Add alerts to cases
+From the Alerts table, you can attach one or more alerts to a case by using the *Add to case* button. From here, you can choose to <<signals-to-new-cases, add the alert to a new case>> or <<signals-to-existing-cases, attach it to an existing one>>. There is no limit to the number of alerts that you can add and you can choose alerts from any rule type - custom or pre-built. If you attach the alert to a case that has been configured to sync its status with associated alerts, the alert's status updates any time the case's status is modified.
+
+NOTE: After you have added an alert to a case, you can only remove it through the the Elastic Security Cases API.
+
+[role="screenshot"]
+image::images/add-alert-to-case.gif[width=80%][height=80%][Shows how to add an alert to a case]
+
+[float]
+[[signals-to-new-cases]]
+==== Add an alert to a new case
+Follow these steps to add an alert to a new case:
+
+. Select **Add to case -> Add to a new case**.
+. In the flyout, give your case a name, add relevant tags, and include a case description.
+. Specify whether you want to sync the statuses of associated alerts. This setting can be toggled on or off at any time, but is enabled by default to allow Cases to control the statuses of attached alerts. If enabled, the alert's status updates any time the case's status is modified.
+. Select a connector. If you've previously added one, that connector displays as the default selection. Otherwise, the default setting is `No connector selected`.
+. Click *Create case* after you've completed all of the required fields. A notification message confirming the case was successfully created displays. Click the link inside the notification or go to the Cases page to view your case.
+
+[role="screenshot"]
+image::images/add-alert-to-new-case.png[Shows how to add an alert to an existing case]
+
+[float]
+[[signals-to-existing-cases]]
+==== Add an alert to an existing case
```

Comment:
Let's make the steps a numbered procedure - (even if it's two steps) for consistency.